config.go

```diff
@@ -18,11 +18,6 @@ type Config struct {
 	// a leader before we attempt an election.
 	ElectionTimeout time.Duration
 
-	// CommitTimeout controls the time without an Apply() operation
-	// before we heartbeat to ensure a timely commit. Due to random
-	// staggering, may be delayed as much as 2x this value.
-	CommitTimeout time.Duration
```

Comment:
Note to James: the new commit index is now just sent at the first opportunity with no artificial delay. I'm not sure why you'd ever need to delay it (it should be low overhead).